Wikipedia/Code/NotificationsCenterViewController.swift

```diff
@@ -64,10 +82,16 @@ final class NotificationsCenterViewController: ViewController {
         viewModel.refreshNotifications()
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        closeSwipeActionsPanelIfNecessary()
+    }
+
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
 
         if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            closeSwipeActionsPanelIfNecessary()
             notificationsView.collectionView.reloadData()
```

Comment:
Just a small update on this - I do notice a bug that might be caused by this:
1. Set reading themes set to default/matches system theme
2. Load notifications center and swipe open a cell
3. Go to iOS settings and switch themes

The screen updates to the new theme but the cell drawer remains open and jumps to another cell, probably because of reuse during the collection view reload.

Reply:
Great catch! I think placing a `closeSwipeActionsPanelIfNecessary()` in `apply(theme:...)` when reloading the data like you mention is sufficient for resolving this, but I'll also add resetting the cell's `foregroundContentContainer`'s transform back to identity in `prepareForReuse` just for extra safety (and to be responsible citizens of cell reuse).